docs/source/guides/distributed_training.mdx

```diff
@@ -183,11 +183,11 @@ Just as for ZeRO-1, it is possible to wrap the optimizer class to make it lazy.
 ```python
 from torch.optim import AdamW
 from optimum.neuron import NeuronAccelerator
-from optimum.neuron.accelerate.utils import ModelParallelismConfig
+from optimum.neuron.models.training.config import TrainingNeuronConfig
 from optimum.neuron.distributed import lazy_load_for_parallelism
 
 tensor_parallel_size = 8
-mp_config = ModelParallelismConfig(
+mp_config = TrainingNeuronConfig(
```

Comment:
Then I would call it `DistributedTrainingNeuronConfig`, but maybe it's too long.